libazureinit/src/unittest.rs

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#[cfg(test)]
+pub(crate) mod unittests {
+    use reqwest::StatusCode;
+
+    use tokio::io::AsyncWriteExt;
+    use tokio::net::TcpListener;
+    use tokio_util::sync::CancellationToken;
+
+    // Returns expected HTTP response for the given status code and body string.
+    pub(crate) async fn get_http_response_payload(
```

Comment:
Since the function doesn't block on any other futures you can declare it as 
```suggestion
    pub(crate) fn get_http_response_payload(
```

Reply:
Done